lib/wanda/operations/catalog/disablepacemaker_v1.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Wanda.Operations.Catalog.DisablePacemakerV1 do
```

Comment:
I guess we will update to `PacemakerDisable`

Reply:
done